### PR TITLE
Add common modules back which should be supported in Azure Functions,…

### LINKED ISSUE #4307: Selecting the wrong module automatically when adding configuration for gradle function project
### Environment:

- OS: Win/Mac
- Plugins Version: Azure java toolkit for IntelliJ: 3.35.0
- IDE Version: IC2019.3.4-jre8/IC20.1 - jre11

### Repro steps:

1. Create a gradle function project
2. Add azure functions run/deploy configuration for the project you created

### Result:

image

image

### Expect:

image

### PR DESCRIPTION
**Pull request by andxu**
… Fix #4307

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/core/FunctionUtils.java [hunk 1]
```diff
@@ -104,10 +105,9 @@ public static boolean isValidStagingFolderPath(String stagingFolderPath) {
     }
 
     public static Module[] listFunctionModules(Project project) {
-        MavenProjectsManager mpm = MavenProjectsManager.getInstance(project);
         final Module[] modules = ModuleManager.getInstance(project).getModules();
         return Arrays.stream(modules).filter(m -> {
-            if (mpm.findProject(m) == null) {
+            if (isModuleInTestScope(m)) {
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/core/FunctionUtils.java:110]: Need test whether module is null here as `GlobalSearchScope.moduleWithLibrariesScope` didn't accept null

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/core/FunctionUtils.java:110]: ModuleManager.getInstance(project).getModules() will not return null modules.
